technical/XSD_EXTENSIONS.md

```diff
@@ -0,0 +1,66 @@
+<!--
+SPDX-FileCopyrightText: 2021 Alliander N.V.
+
+SPDX-License-Identifier: CC-BY-4.0
+-->
+
+## XSD Extensions
+For generating the SCL classes, we use the official IEC SCL XSD schemas. These are stored in the [XSD repository](https://github.com/com-pas/compas-scl-xsd). Every week, different Github Actions check for newer version of these XSD schemas. In case newer versions are available, the Github Actions fail and people will get notifications.
+
+In case organizations need some specific information in their SCL files, so called "XSD extensions" can be used. These are separate XSD files containing additional fields or other information that can be used.
+
+### Using your own XSD extensions
+Because the SCL XSD schemas are stored in a separate repository and are available as a separate [library](https://github.com/com-pas/compas-scl-xsd/packages/817016), every project that wants to use it needs to include the dependency and unpack the schemas. When doing this, a project is free of how to use them.
+
+XSD schema extensions can be included in the base SCL XSD schemas by using the [xsd:import](https://www.w3.org/TR/xmlschema-1/#composition-schemaImport) (different namespace) or the [xsd:include](https://www.w3.org/TR/xmlschema-1/#compound-schema) (same or no namespace) statement.
+
+#### Project specific XSD extensions
+Project specific XSD schemas can be defined within the project itself. By using the previous import and include statements, they can be included and used. **This means that during unpacking the base SCL XSD library, this include/import/use process should be added before generating classes**.
+
+#### CoMPAS-broad XSD extensions
+If we need CoMPAS-broad specific XSD schemas, these can be added to the [XSD repository](https://github.com/com-pas/compas-scl-xsd). **This means that during the build of the SCL XSD library, this include/import/use process should be done**.
```

Comment:
We need a CoMPAS broad extension e.g. for filename and SCL file type.

Reply:
Do you want these examples to be included? @Sander3003

Reply:
These examples can be added to me more clear.

Reply:
Added